storage/database/src/main/java/de/danoeh/antennapod/storage/database/PodDBAdapter.java

```diff
@@ -50,7 +50,7 @@ public class PodDBAdapter {
 
     private static final String TAG = "PodDBAdapter";
     public static final String DATABASE_NAME = "Antennapod.db";
-    public static final int VERSION = 2060000;
+    public static final int VERSION = 3010000;
```

Comment:
@ByteHamster I chose this DB version arbitrarily, I'm not sure if there is a convention to how the version is integrated. Let me know if this should be changed before merge.

Actually, I changed it to `3010000` because it was on `3000000` before I made my most recent changes. I must have incremented it in an earlier change. We might want to roll it back to `3000000`.

Reply:
I think we will merge this for AntennaPod 3.1 (I want to release 3.0 as soon as possible, so I'm trying to not add too many new features now that could delay a release). The version code is exactly what we need for 3.1 :)

Reply:
Fantastic, a happy accident :grinning: